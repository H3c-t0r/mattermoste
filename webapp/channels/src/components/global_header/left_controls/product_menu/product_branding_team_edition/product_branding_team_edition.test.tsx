// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import {shallow} from 'enzyme';
import React from 'react';

import ProductBrandingTeamEdition from './product_branding_team_edition';

describe('components/ProductBrandingTeamEdition', () => {
    test('should show correct product branding for team edition', () => {
        const wrapper = shallow(
            <ProductBrandingTeamEdition/>,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
