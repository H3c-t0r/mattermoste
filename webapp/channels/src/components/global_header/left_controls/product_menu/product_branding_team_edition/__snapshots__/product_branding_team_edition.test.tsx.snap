// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ProductBrandingTeamEdition should show correct product branding for team edition 1`] = `
<ProductBrandingTeamEditionContainer
  tabIndex={0}
>
  <MattermostIcon
    size={24}
  />
  <Heading
    element="h1"
    margin="none"
    size={200}
  >
    Mattermost
  </Heading>
  <Badge>
    FREE EDITION
  </Badge>
</ProductBrandingTeamEditionContainer>
`;
